projects/packages/jetpack-mu-wpcom/src/features/pages/js/pages-homepage-connection-banner.js

```diff
@@ -8,6 +8,21 @@
 
 /* global jQuery */
 ( function ( $ ) {
+	const trackEvent = ( eventName, eventProperties = {} ) => {
```

Comment:
We have a function for this here: https://github.com/Automattic/jetpack/blob/45f8c2f30d6464e5bee9a2930ee97481fcc40616/projects/packages/jetpack-mu-wpcom/src/common/tracks.js

Do you think it's possible to re-use it in this context?